osp/core/ontology/yml/yml_parser.py

```diff
@@ -404,8 +406,51 @@ def _set_inverse(self, entity_name, entity_doc):
             (iri, rdflib.OWL.inverseOf, inverse_iri)
         )
 
-    def _check_default_rel(self, entity_name, entity_doc):
-        """Check if the given relationship the default
+    def _assert_default_relationship_occurance(self):
```

Comment:
Occurrence is the correct way to spell this word, meaning 'an event that happened.'

Occurance is one of the most popular ways that this world is misspelt.

https://hinative.com/en-US/questions/1554753

Is this true?

Reply:
I will update it to occurrence

Reply:
True: https://www.dict.cc/?s=occurrence

But I am happy to be one out of millions :)